libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -639,6 +639,15 @@ const AP_Param::GroupInfo NavEKF3::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("GSF_RST_MAX", 60, NavEKF3, _gsfResetMaxCount, 2),
 
+    // @Param: AFFINITY
+    // @DisplayName: EKF3 Sensor Affinity Options
+    // @Description: These options control the affinity between sensor instances and EKF cores
+    // @User: Advanced
+    // @Bitmask: 0:EnableGPSAffinity,1:EnableBaroAffinity,2:EnableCompassAffinity,3:EnableAirspeedAffinity
+    AP_GROUPINFO("AFFINITY", 61, NavEKF3, _affinity, 0),
```

Comment:
Unless we have tested the ability to cope with changes to these settings without rebooting, then I would recommend labelling this as

// @RebootRequired: True

Reply:
i don't think a reboot is required, because the sensor selection functions which assign the sensors to respective cores first assign the primary instance and then change the assignment only if affinity for that respective sensor is available. This happens whenever we update the filter. So if affinity is turned off mid-flight after being on (or vice-versa) we would still be using the primary instance of that sensor for all cores (or affinity based assignment). Maybe @tridge can better judge this ?